src/hooks/useAudio.js

```diff
@@ -1,18 +1,29 @@
+// @ts-check
 import { useEffect } from 'react'
 import { useState } from 'react'
 
 export default function useAudio(url) {
   const [song, setSong] = useState(new Audio(url))
   const [isPlaying, setIsPlaying] = useState(false)
-
+  const [vol, setVol] = useState(0.5)
```

Comment:
Why not like that?
```suggestion
  const [volume, setVolume] = useState(0.5)
```

Reply:
Here it seems I don't quite understand it yet. At the moment my function calls volume can i then use the name twice? I think no right? So it is the consequence of moderate naming on my part.